modules/graph/tools/fragment_loader_impl.h

```diff
@@ -50,34 +50,51 @@ ObjectID load_graph(Client& client, grape::CommSpec& comm_spec,
                          options.generate_eid, options.retain_oid,
                          options.local_vertex_map, options.compact_edges,
                          options.use_perfect_hash);
-
   MPI_Barrier(comm_spec.comm());
   auto fn = [&]() -> boost::leaf::result<ObjectID> {
     return loader.LoadFragmentAsFragmentGroup(options.efiles, options.vfiles);
   };
-
   auto wholefn = [&]() -> boost::leaf::result<ObjectID> {
     ObjectID frag = InvalidObjectID();
     BOOST_LEAF_ASSIGN(frag, loader.LoadFragment());
     loader_t adder(client, comm_spec, options.efiles, options.vfiles,
                    options.directed, options.generate_eid, options.retain_oid);
     return adder.AddLabelsToFragmentAsFragmentGroup(frag);
   };
-
   auto stepbystepfn = [&]() -> boost::leaf::result<ObjectID> {
     ObjectID frag = InvalidObjectID();
     BOOST_LEAF_ASSIGN(frag, loader.LoadFragment());
+    int64_t vlabel_num = 0;
     for (auto vfile : options.vfiles) {
-      loader_t adder(client, comm_spec, std::vector<std::string>{},
-                     std::vector<std::string>{vfile}, options.directed,
-                     options.generate_eid, options.retain_oid);
-      BOOST_LEAF_ASSIGN(frag, adder.AddLabelsToFragment(frag));
+      std::vector<std::string> sub_files;
+      boost::split(sub_files, vfile, boost::is_any_of(";"));
```

Comment:
emm, let's talk about that design later.
I think it's not the best way to use incremental adding method